happi/utils.py

```diff
@@ -20,3 +23,31 @@ def get_happi_entry_value(entry, key, search_extraneous=True):
     if not value:
         raise ValueError('Invalid Key for Device.')
     return value
+
+
+def is_number(str_value):
+    """
+    Checks if it is a valid float number
+    """
+    try:
+        float(str_value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_a_range(str_value):
+    """
+    Checks to see if it is a range.
+    It needs to have two valid values separated by a comma.
+    Ex:
+        1,100
+        -2,8
+        100,2
+    """
+    if ',' in str_value:
+        start, stop = str_value.split(',')
+        if is_number(start) and is_number(stop):
+            return True
+        else:
+            logger.error("Possibly provided invalid numbers for a range")
```

Comment:
We probably want to return `False` at the bottom here instead of `None`

Reply:
Yes that's right, thank you for catching that. Fixed it in cf7fd10

Reply:
Still one more hole here, where were return `None` if the initial `if ',' in str_value` is `False`

Reply:
Ohh sorry about that, added it in bf1c57f .